Cargo.toml

```diff
@@ -150,15 +150,15 @@ openssl = { version = "0.10", features = ["vendored"] }
 # Using the 3.3 version of the FVM because the newer ones update the IPLD dependencies
 # to version which are different than the ones in the builtin-actors project, and since
 # they are 0.x cargo cannot upgrade them automatically, which leads to version conflicts.
-fvm = { version = "~3.2", default-features = false }     # no opencl feature or it fails on CI
-fvm_shared = { version = "~3.2", features = ["crypto"] }
-fvm_sdk = { version = "~3.2" }
+fvm = { version = "4.0.0", default-features = false }     # no opencl feature or it fails on CI
+fvm_shared = { version = "4.0.0", features = ["crypto"] }
+fvm_sdk = { version = "4.0.0" }
```

Comment:
I wonder if it's worth limiting the updates to eg. `"~4.0"` so we accept patches but not new features automatically? Based on prior experience they can be breaking even on a minor release.

Reply:
Probably not atm, will need to update this to support the pluggable syscalls for custom kernel